fastembed/postprocess/muvera.py

```diff
@@ -336,3 +349,11 @@ def process(
 
         # Concatenate results from all R_reps projections into final FDE
         return np.concatenate(output_vectors)
+
+
+if __name__ == "__main__":
+    v_arrs = np.random.randn(10000, 100, 128)
+    muvera = Muvera(128, 4, 8, 20, 42)
+
+    for v_arr in v_arrs:
+        muvera.process(v_arr)  # type: ignore
```

Comment:
Do we want to keep this for more benchmarking? Afaik, nothing else in fastembed can be executed like this.